.github/workflows/test.yml

```
@@ -0,0 +1,130 @@
+          - '26.3'
+          - '27.1'
+          - '27.2'
+          - '28.1'
+          - '28.2'
+          - '29.1'
+          - '29.2'
+          - '29.3'
+          - 'snapshot'
+      fail-fast: false
+
+    steps:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 27: did not find expected '-' indicator

(syntax-check)